docker/webui/Dockerfile

```diff
@@ -1,14 +1,14 @@
+#!BuildTag: openqa_webui
 FROM opensuse/leap:15.2
 LABEL maintainer Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>, wnereiz <wnereiz@eienteiland.org>, Sergio Lindo Mansilla <slindomansilla@suse.com>
 LABEL version="0.3"
 
-RUN zypper ar -f obs://devel:openQA/openSUSE_Leap_15.2 openQA && \
-    zypper ar -f obs://devel:openQA:Leap:15.2/openSUSE_Leap_15.2 openQA-perl-modules && \
+RUN zypper ar -p 95 -f http://download.opensuse.org/repositories/devel:openQA/openSUSE_Leap_15.2 devel_openQA && \
+    zypper ar -p 90 -f http://download.opensuse.org/repositories/devel:openQA:Leap:15.2/openSUSE_Leap_15.2 devel_openQA_Leap && \
```

Comment:
> same as in [#3544 (comment)](https://github.com/os-autoinst/openQA/pull/3544#issuecomment-725666426)
> 
> Regarding the obs://… format. You could try to create a ticket on github.com/openSUSE/open-build-service/issues/new and report that. Just 6 days ago I reported [openSUSE/open-build-service#10396](https://github.com/openSUSE/open-build-service/issues/10396) about another parsing error which was promptly fixed.

Can we put a ticket/bug ref in a comment maybe? Or mention it in progress?

Reply:
I reported the problem with the obs:// format in this bug https://github.com/openSUSE/open-build-service/issues/10438

Reply:
awesome! Thanks.